pkg/systemd/services.html

```diff
@@ -56,7 +56,10 @@
         <button class="btn btn-default" role="tab" data-pattern="\.timer$" translatable="yes">Timers</button>
         <button class="btn btn-default" role="tab" data-pattern="\.path$" translatable="yes">Paths</button>
       </div>
-      <button id="create-timer" data-toggle="#timer-dialog" data-target="#timer-dialog" class="btn btn-primary pull-right" translate>Create Timer</button>
+      <div class="filter-group">
+        <button id="create-timer" data-toggle="#timer-dialog" data-target="#timer-dialog" class="btn btn-primary" translate>Create Timer</button>
+        <input type="search" id="services-text-filter" class="form-control" aria-label="Filter" placeholder="Filter by id or description..." translate="placeholder"/>
```

Comment:
placeholder needs i18n. Also, shouldn't "id" be written as "ID"? 

systemd calls that "unit name". As we want to avoid jargon as much as possible, "Filter by name or description..." would be better. But as we already show "ID" in the table, let's postpone that part for another time.

Reply:
> Also, shouldn't "id" be written as "ID"?

In the table columns it is called `Id`. Design has `id`. 
But agree that `name` would be more suitable, but lets to this in another PR :)

Reply:
> placeholder needs i18n

`translate="placeholder"` is already there